dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -102,6 +105,28 @@ public List<ErrorRest> validate(SubmissionService submissionService, InProgressS
         return getErrors();
     }
 
+    private boolean isValidComplexDefinitionMetadata(DCInput input, List<MetadataValue> mdv) {
+        if (input.getInputType().equals("complex")) {
+            int complexDefinitionIndex = 0;
+            for (String complexDefinitionInputName : input.getComplexDefinition().getInputs().keySet()) {
```

Comment:
use abbrev. for `input.getComplexDefinition().getInputs()`?

Reply:
refactored